### ISSUE TITLE
traefik whoami routers - purpose?

### ISSUE THREAD

**Issue by dajuno** [MaRDI4NFDI/portal-compose#132]
1. It is unclear to me what the purpose of some of the routers is, defined in [`traefik/conf/`](https://github.com/MaRDI4NFDI/portal-compose/tree/main/traefik/conf). In particular [`https-redirect.yml`](https://github.com/MaRDI4NFDI/portal-compose/blob/main/traefik/conf/https-redirect.yml) seems to apply to any host and redirect it (via https) to the whoami service (-> https://whoami.mardi4nfdi.de/). Why, and what hosts/urls does this actually apply to? How come the high priority of 100 does not cause this router to take over all host queries and redirect everything to whoami?

2. If this router just redirects all http requests to https (I don't see why it routes to whoami though), would it make sense to just use this simple entryPoint setting in `traefik.yml`?

```yaml
entryPoints:
    web:
        address: :80
        # redirect to https
        http:
            redirections:
                entryPoint:
                    to: websecure
                    scheme: https
    websecure:
        address: :443
```

@physikerwelt

**Comment by dajuno** [comment]
> @dajuno can you prepare a pull for the removal of whoami?

will do. should the whoami container be removed completely or just the traefik router?

**Comment by dajuno** [comment]
forgot to remove the whoami container from the dev build, fixed by #136
